docker-compose.yml

```diff
@@ -6,62 +6,58 @@ version: "3.9"
 
 services:
   covalent-data:
-    image: "${DATA_REPO}"
+    image: "public.ecr.aws/covalent/covalent_data"
     ports:
-      - "8004:8004"
+      - "80:8004"
     working_dir: /data
     networks:
-     covalent:
+      covalent:
 
   covalent-dispatcher:
-    image: "${DISPATCHER_REPO}"
+    image: "public.ecr.aws/covalent/covalent_dispatcher"
     ports:
-      - "8002:8002"
+      - "80:8002"
     working_dir: /dispatcher
     networks:
-     covalent:
+      covalent:
 
   covalent-queuer:
-    image: "${QUEUER_REPO}"
+    image: "public.ecr.aws/covalent/covalent_queuer"
     ports:
-      - "8001:8001"
+      - "80:8001"
     working_dir: /queuer
     networks:
       covalent:
 
   # Queue-consumer uses the same Dockerfile configurations as the Queuer
   covalent-queue-consumer:
-    image: "${QUEUE_CONSUMER_REPO}"
+    image: "public.ecr.aws/covalent/covalent_queuer"
     ports:
-      - "3000:8001"
+      - "80:8001"
     working_dir: /queuer
     networks:
       covalent:
 
   covalent-results:
-    image: "${RESULTS_REPO}"
+    image: "public.ecr.aws/covalent/covalent_results"
     ports:
-      - "8006:8006"
+      - "80:8006"
     working_dir: /results
     networks:
-     covalent:
+      covalent:
 
   covalent-runner:
-    image: "${RUNNER_REPO}"
+    image: "public.ecr.aws/covalent/covalent_runner"
     ports:
-      - "8003:8003"
+      - "80:8003"
     working_dir: /runner
     networks:
       covalent:
 
   covalent-ui:
-    image: "${UI_REPO}"
+    image: "public.ecr.aws/covalent/covalent_ui"
     ports:
-      - 3001:8004
-      - 8005:8005
-    environment:
-      UI_SVC_HOST: 0.0.0.0
-      UI_SVC_PORT: 8004
+      - "80:8004"
```

Comment:
@Emmanuel289 can you confirm that this piece looks OK from your side?

Reply:
yeah, this looks good